src/AuthProvider.tsx

```diff
@@ -167,7 +167,7 @@ export const AuthProvider = (props: AuthProviderProps): JSX.Element => {
 
     // register to userManager events
     useEffect(() => {
-        if (!userManager) return undefined;
+        if (!userManager || !userManager.events) return undefined;
```

Comment:
if this wasn't added, when `implementation` is set to `null` in provider, some tests would fail due to the fact that they are trying to call `userManager.events.addUserLoaded` and `events` is `null`

Reply:
We need to fix this in the tests then, the "real" UserManager does always have events...

Reply:
will look into it shortly

Reply:
oh it was my bad, that error would not make the test fail lol